controls/1_1_filesystem_configuration.rb

```diff
@@ -291,116 +346,116 @@
   tag level: 1
 
   describe mount('/dev/shm') do
     its(:options) { should include 'nosuid' }
   end
 end
 
-control 'cis-dil-benchmark-1.1.17' do
+control 'cis-dil-benchmark-1.1.21' do
   title 'Ensure noexec option set on /dev/shm partition'
   desc  "The noexec mount option specifies that the filesystem cannot contain executable binaries.\n\nRationale: Setting this option on a file system prevents users from executing programs from shared memory. This deters users from introducing potentially malicious software on the system."
   impact 1.0
 
-  tag cis: 'distribution-independent-linux:1.1.17'
+  tag cis: 'distribution-independent-linux:1.1.21'
   tag level: 1
 
   describe mount('/dev/shm') do
     its(:options) { should include 'noexec' }
   end
 end
 
-control 'cis-dil-benchmark-1.1.18' do
+control 'cis-dil-benchmark-1.1.22' do
   title 'Ensure nodev option set on removable media partitions'
   desc  "The nodev mount option specifies that the filesystem cannot contain special devices.\n\nRationale: Removable media containing character and block special devices could be used to circumvent security controls by allowing non-root users to access sensitive device files such as /dev/kmem or the raw disk partitions."
   impact 0.0
 
-  tag cis: 'distribution-independent-linux:1.1.18'
+  tag cis: 'distribution-independent-linux:1.1.22'
   tag level: 1
 
-  describe 'cis-dil-benchmark-1.1.18' do
+  describe 'cis-dil-benchmark-1.1.22' do
     skip 'Not implemented'
   end
 end
 
-control 'cis-dil-benchmark-1.1.19' do
+control 'cis-dil-benchmark-1.1.23' do
   title 'Ensure nosuid option set on removable media partitions'
   desc  "The nosuid mount option specifies that the filesystem cannot contain setuid files.\n\nRationale: Setting this option on a file system prevents users from introducing privileged programs onto the system and allowing non-root users to execute them."
   impact 0.0
 
-  tag cis: 'distribution-independent-linux:1.1.19'
+  tag cis: 'distribution-independent-linux:1.1.23'
   tag level: 1
 
-  describe 'cis-dil-benchmark-1.1.19' do
+  describe 'cis-dil-benchmark-1.1.23' do
     skip 'Not implemented'
   end
 end
 
-control 'cis-dil-benchmark-1.1.20' do
+control 'cis-dil-benchmark-1.1.24' do
   title 'Ensure noexec option set on removable media partitions'
   desc  "The noexec mount option specifies that the filesystem cannot contain executable binaries.\n\nRationale: Setting this option on a file system prevents users from executing programs from the removable media. This deters users from being able to introduce potentially malicious software on the system."
   impact 0.0
 
-  tag cis: 'distribution-independent-linux:1.1.20'
+  tag cis: 'distribution-independent-linux:1.1.24'
   tag level: 1
 
-  describe 'cis-dil-benchmark-1.1.20' do
+  describe 'cis-dil-benchmark-1.1.24' do
     skip 'Not implemented'
   end
 end
 
-control 'cis-dil-benchmark-1.1.21' do
+control 'cis-dil-benchmark-1.1.25' do
```

Comment:
We should always be careful with changing the control ids. I am aware that CIS is doing it, I think we should switch to our own number scheme going forward and attach the cis control id only as a tag

Reply:
Do the control numbers only change when CIS releases a newer version of their benchmarks? If so, could you use git tags to track the CIS profile versions in order to continue using the current control naming scheme? Just a thought, not sure what you have in mind going forward.